### PR TITLE
add support for invite members after channel creation

### PR DESCRIPTION
**Pull request by thesyncim**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: channel.go [hunk 1]
```diff
@@ -189,6 +189,21 @@ func (ch *Channel) AddModerators(userIDs ...string) error {
 	return ch.client.makeRequest(http.MethodPost, p, nil, data, nil)
 }
 
+// InviteMembers invites users with given IDs to the channel
+func (ch *Channel) InviteMembers(userIDs ...string) error {
+	if len(userIDs) == 0 {
+		return errors.New("user IDs are empty")
```
Comment (pascaldekloe) [inline channel.go]: `return nil`

Comment (thesyncim) [reply channel.go]: im following the same convention used here https://github.com/GetStream/stream-chat-go/pull/24/files#diff-861e77702f49a71d32be4b3c7600818fR210

**Comment by thesyncim** [comment]
> Can you also update interface at stream_chat.go?

yes. we should add a check var _ ChatIntrr

File: channel.go (hunk 1 above)
Comment (pascaldekloe) [reply channel.go]: We define conventions now? 😆

**Pull request merged**